src-cljs/atom_parinfer/core.cljs

```diff
@@ -273,6 +273,41 @@
 
 (add-watch *editor-states :toggle-status-classes toggle-status-classes!)
 
+;;------------------------------------------------------------------------------
+;; Error Markers
+;;------------------------------------------------------------------------------
+
+(def error-marker-class "parinfer-error-marker")
+
+(defn add-error-marker
+  [js-editor start-row js-error]
+  (let [row (+ (oget js-error "lineNo") start-row)
+        col (oget js-error "x")
+        marker (ocall js-editor "markBufferRange"
+                 (array (array row col)
+                        (array row (inc col)))
+                 (js-obj "invalidate" "never"))
+        decorator (ocall js-editor "decorateMarker" marker
+                    (js-obj "type" "highlight"
+                            "class" error-marker-class))]
+    (js/console.log "position" row col)
+    (js/console.log "marker" marker)
+    (js/console.log "decorator" decorator)))
+
+
+(defn get-previous-error-markers
+  [js-editor start-row end-row]
+  (let [markers (ocall js-editor "findMarkers")]
+                  ; (js-obj "startBufferRow" start-row
+                  ;         "endBufferRow" end-row))]
+    (js/console.log "markers" markers)
+    markers))
+
+(defn clear-previous-error-markers
+  [js-editor start-row end-row js-error]
+  (doseq [marker (get-previous-error-markers js-editor start-row end-row)]
+    ; (ocall marker "destroy"))
```

Comment:
not destroying them yet since I'm not sure how to determine which markers _we_ created